core/src/main/java/org/mule/runtime/core/internal/construct/DefaultFlowBuilder.java

```diff
@@ -33,6 +33,7 @@
 import org.mule.runtime.core.api.source.MessageSource;
 import org.mule.runtime.core.internal.exception.MessagingException;
 import org.mule.runtime.core.internal.processor.strategy.DirectProcessingStrategyFactory;
+import org.mule.runtime.core.internal.processor.strategy.TransactionAwareProactorStreamEmitterProcessingStrategyFactory;
 import org.mule.runtime.core.internal.processor.strategy.TransactionAwareProactorStreamWorkQueueProcessingStrategyFactory;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused import 'org.mule.runtime.core.internal.processor.strategy.TransactionAwareProactorStreamWorkQueueProcessingStrategyFactory'. ([squid:UselessImportCheck](https://sonarqube.kbuild.msap.io/coding_rules#rule_key=squid%3AUselessImportCheck))

[See it in SonarQube](https://sonarqube.kbuild.msap.io/project/issues?id=mule-runtime-mule&pullRequest=7657&issues=AWjJeTB0otIXnmsUTyuN&open=AWjJeTB0otIXnmsUTyuN)

[ISSUE_KEY]: AWjJeTB0otIXnmsUTyuN